sources/USAGE.md

```diff
@@ -0,0 +1,75 @@
+# Usage
+
+## Library
+
+Since version 2.2.0, Alt-Ergo's library is also compiled and installed. A small example using the API is given [here](examples/lib_usage.ml). You can also found some documentation [here](https://ocamlpro.github.io/alt-ergo/dev/alt-ergo-lib/index.html)
```

Comment:
```suggestion
Since version 2.2.0, Alt-Ergo's library is also compiled and installed. A small example using the API is given [here](examples/lib_usage.ml). You can also find some documentation [here](https://ocamlpro.github.io/alt-ergo/dev/alt-ergo-lib/index.html)
```